app/rtc_s1.py

```diff
@@ -55,16 +56,31 @@ def _update_mosaic_boundaries(mosaic_geogrid_dict, geogrid):
         mosaic_geogrid_dict['yf'] = yf
     if 'dx' not in mosaic_geogrid_dict.keys():
         mosaic_geogrid_dict['dx'] = geogrid.spacing_x
-    else:
-        assert(mosaic_geogrid_dict['dx'] == geogrid.spacing_x)
+    # else:
+    #     assert(mosaic_geogrid_dict['dx'] == geogrid.spacing_x)
     if 'dy' not in mosaic_geogrid_dict.keys():
         mosaic_geogrid_dict['dy'] = geogrid.spacing_y
-    else:
-        assert(mosaic_geogrid_dict['dy'] == geogrid.spacing_y)
+    # else:
+    #     assert(mosaic_geogrid_dict['dy'] == geogrid.spacing_y)
+
+
+
+
+
+
+
+    # TODO fix this. The mosaic EPSG should not come from the
+    # first burst
     if 'epsg' not in mosaic_geogrid_dict.keys():
         mosaic_geogrid_dict['epsg'] = geogrid.epsg
-    else:
-        assert(mosaic_geogrid_dict['epsg'] == geogrid.epsg)
+    # else:
+    #     assert(mosaic_geogrid_dict['epsg'] == geogrid.epsg)
+
+
+
```

Comment:
@gshiroma 
This is enough to fix the RTC crush on the SDS side. Please decouple this from the layover related changes. We need the EPSG fix today.